nss-day-cohort-45/hello-world-column-counts
# Rh japan updates

## changes made
1) matched background color page wide
2) added grayscale on hover to flag images
3) gave flag borders rounded edges

## steps to test
1) serve it up
2) check it out
3) ensure all changes discussed have been made
